komodo/insert_proposals.py

```diff
@@ -18,22 +20,41 @@
 )
 
 
-def recursive_update(left, right):
-    if right is None:
+def recursive_update(
+    to_be_upgraded: Mapping[str, Union[str, Mapping[str, str]]],
+    upgrades: Optional[Mapping[str, Union[str, Mapping[str, str]]]],
+) -> None:
+    """Updates release in place with upgrades/additions of packages from upgrades.
+        Takes format:
```

Comment:
misaligned whitespace before `Takes`. Could that create issues if this is viewed through sphinx?

Reply:
Formatting docstrings is a nightmare. It wraps if I have it on the same indentation.